app/views/ctc/ctc_pages/home.html.erb

```diff
@@ -30,8 +30,8 @@
           </p>
         <% end %>
           <% if open_for_ctc_intake? %>
-            <p class="spacing-below-35"><%= t("views.ctc_pages.home.subheader.claim") %></p>
-            <p><%= t("views.ctc_pages.home.subheader.time_estimate") %></p>
+            <p class="spacing-below-35"><%= Flipper.enabled?(:eitc) ? t("views.ctc_pages.home.subheader.claim.eitc_on") : t("views.ctc_pages.home.subheader.claim.eitc_off") %></p>
```

Comment:
should probably rebase and change these to `open_for_eitc_intake?` (which respects both the `:eitc_beta` and the feature flag)